README.md

```diff
@@ -1,6 +1,6 @@
 # Codacy Visual Studio Code extension
 
-The Codacy extension for Visual Studio Code helps you review and manage the issues found by Codacy directly within Visual Studio Code. It notifies you whether a pull request is up to standards by highlighting problematic code patterns and displaying code quality metrics.
+The Codacy extension for Visual Studio Code helps you review and manage the issues found by Codacy without leaving the editor. It informs you about the code quality changes introduced by the pull request you're working on, as well as the status of other open pull requests and analyzed branches.
```

Comment:
@alerizzo, as discussed on Slack, here's a quick edit highlighting that the branch doesn't need to be associated with an open PR. IMO, we can skip this, as it adds information that may not be very relevant at this point in the user's journey, and it's clarified [on Codacy's docs](https://github.com/codacy/docs/pull/1936/files#diff-4844371c8802185733a8fe9027911aa34832f676dbe2f21fbd25bc401dc61a0cR65). Thoughts?

```suggestion
The Codacy extension for Visual Studio Code helps you review and manage the issues found by Codacy without leaving the editor. It informs you about the code quality changes introduced by the pull request you're working on, as well as the status of other open pull requests and of branches analyzed by Codacy that are not associated with any pull request.
```

Reply:
Like it1